charts/merlin/README.md

```diff
@@ -329,6 +329,8 @@ The following table lists the configurable parameters of the Merlin chart and th
 | mlflowExternalPostgresql.username | string | `"mlflow"` | External postgres database user |
 | mlp.enabled | bool | `true` |  |
 | mlp.environmentConfigSecret.name | string | `""` |  |
+| rules.apiPrefixRegex | string | `".+"` |  |
+| rules.enabled | bool | `false` | Enable this to provision Oathkeeper Rule CRDs for Merlin API endpoints |
```

Comment:
can we have a prefix for rules? it will be unclear what are the rules about outside of this MR context